filament/backend/src/opengl/OpenGLProgram.h

```diff
@@ -38,6 +38,24 @@ namespace filament::backend {
 
 class OpenGLDriver;
 
+struct PushConstantBundle {
+    uint8_t fragmentStageOffset = 0;
+    utils::FixedCapacityVector<std::pair<GLint, ConstantType>> const* constants = nullptr;
```

Comment:
I think you can even make this a reference

Reply:
used two slices instead.